paasta_tools/flink_tools.py

```diff
@@ -183,6 +183,22 @@ def get_flink_config(
             raise
 
 
+def get_flink_metadata(
+        kube_client: KubeClient, service: str, instance: str
+) -> Optional[Mapping[str, any]]:
+    try:
+        co = kube_client.custom.get_namespaced_custom_object(
+            **flink_custom_object_id(service, instance)
+        )
+        metadata = co.get("metadata")
+        return metadata
+    except ApiException as e:
+        if e.status == 404:
+            return None
+        else:
+            raise
```

Comment:
raise the original exception here, we don't want to discard the reason this failed

Reply:
but that's what `raise` with no argument does, inside `except` clause. Did I miss something?

Reply:
you're right! TIL